presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestIntervalDayToSecondVarianceAggregation.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation;
+
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.block.BlockBuilder;
+import com.facebook.presto.spi.block.BlockBuilderStatus;
+import com.facebook.presto.spi.type.StandardTypes;
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.math3.stat.descriptive.moment.Variance;
+
+import java.util.List;
+
+import static com.facebook.presto.spi.type.IntervalDayTimeType.INTERVAL_DAY_TIME;
+
+public class TestIntervalDayToSecondVarianceAggregation
+        extends AbstractTestAggregationFunction
+{
+    @Override
+    public Block[] getSequenceBlocks(int start, int length)
+    {
+        BlockBuilder blockBuilder = INTERVAL_DAY_TIME.createBlockBuilder(new BlockBuilderStatus(), length);
+        for (int i = start; i < start + length; i++) {
+            INTERVAL_DAY_TIME.writeLong(blockBuilder, i);
+        }
+        return new Block[] {blockBuilder.build()};
+    }
+
+    @Override
+    public Number getExpectedValue(int start, int length)
+    {
+        if (length < 2) {
+            return null;
+        }
+
+        double[] values = new double[length];
+        for (int i = 0; i < length; i++) {
+            values[i] = start + i;
+        }
+
+        Variance variance = new Variance();
+        return variance.evaluate(values);
+    }
+
+    @Override
+    protected String getFunctionName()
+    {
+        return "variance";
```

Comment:
@maciejgrzybek `variance` is alias of `var_samp`, and `variance` is used in other tests such as [TestLongVarianceAggregation.java#L59](https://github.com/facebook/presto/blob/master/presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestLongVarianceAggregation.java#L59)